### PR TITLE
feat: filter functionality index page

### PR DESCRIPTION
**Pull request by joannaong**
## Description
- show selected filters beside results count
- clear all filters
- filters button in nav
- store layout in state
- scaffold grid and list card layouts
- fix results per page component

### CODE REVIEW

File: packages/fe/modules/form/store/index.js [hunk 1]
```diff
@@ -43,8 +43,8 @@ const actions = {
     const fields = CloneDeep(getters.fields.filter(obj => obj.formId === formId))
     fields.forEach((field) => {
       dispatch('updateFormField', Object.assign(field, {
-        value: incoming[field.model_key],
-        originalValue: incoming[field.model_key],
+        value: field.type === 'select' ? 0 : incoming[field.model_key],
+        originalValue: field.type === 'select' ? 0 : incoming[field.model_key],
```
Comment (timelytree) [inline packages/fe/modules/form/store/index.js:47]: Ah you just reminded me to port over the new forms module! I'll let ya know when it's in :)

File: packages/fe/modules/search/plugins/index.js [hunk 2]
```diff
@@ -29,4 +29,13 @@ export default async function ({ app, store }, inject) {
...
   })
+  inject('clearAllFilters', (exception) => {
+    app.$filter.clearAll(exception)
+  })
+  inject('clearAll', () => {
+    app.$search.clearSearchQuery()
+    app.$filter.clearAll()
+    app.$sort.clearQuery()
+    app.$limit.clearQuery()
+  })
```
Comment (timelytree) [inline packages/fe/modules/search/plugins/index.js:40]: Let's rename this to something more specific since this becomes a global helper!

**Review by timelytree — CHANGES_REQUESTED** [review]

File: packages/fe/store/datasets.js [hunk 3]
```diff
@@ -18,7 +14,8 @@ const state = () => ({
   basicStats: false,
   filters: false,
   sort: false,
-  limit: false
+  limit: false,
```
Comment (timelytree) [inline packages/fe/store/datasets.js:17]: Looks like you have 2 limit values set in the store?

**Review by timelytree — CHANGES_REQUESTED** [review]

File: packages/fe/modules/search/plugins/index.js (hunk 2 above)
Comment (joannaong) [reply packages/fe/modules/search/plugins/index.js]: done

File: packages/fe/store/datasets.js (hunk 3 above)
Comment (joannaong) [reply packages/fe/store/datasets.js:16]: done

**Pull request merged**